lib/transaction_commit.c

```diff
@@ -265,6 +265,11 @@ xbps_transaction_commit(struct xbps_handle *xhp)
 		    "%s\n", strerror(rv));
 		goto out;
 	}
+
+	if (xhp->flags & XBPS_FLAG_DOWNLOAD_ONLY) {
+		goto out;
+	}
+
```

Comment:
I'm pretty sure I use the UNPACK_ONLY code in this function, and use this to shortcut and get out.

Reply:
Now that you mention it, @pullmoll, had we pushed with 0x00003000 this would have broken unpacking.

Reply:
*Always be careful with the clamps. You may short circuit something you did not intend.*